### ISSUE TITLE
GameGuru MAX - UI Issue - Character Creator Custom Area Too Small

### ISSUE THREAD

**Issue by ArgentArts** [TheGameCreators/GameGuruRepo#2128]
Why is this area so small?

![cc](https://user-images.githubusercontent.com/36735507/154144393-df807141-633a-4b55-b309-3d4089af9d13.png)

My screen resolution is 3440 x 1440. Shouldn't this area expand so I can see all the heads instead of only a single row and barely part of another?

**Comment by ZakJudges** [comment]
Hi @ArgentArts, I've added a fix for this which you should see in Friday's update. Unfortunately I do not have an ultra-wide monitor to test it. Would you be able to give feedback after you have tried it? :)

**Comment by ArgentArts** [comment]
Definitely. I'll let you know Friday or so. Thanks for letting me know.

**Comment by ArgentArts** [comment]
Okay, so, it is larger, thank you. But let's look at this a moment:

![cc1](https://user-images.githubusercontent.com/36735507/156883007-482bf645-e0a9-4868-acb6-f2c814dc542e.png)

Notice all the extra free room at the bottom? Why not have all those closed panels slid to the bottom of the screen so all the heads can be seen? Then there would be no scrolling at all. And with the other selections (other than heads), there would either be no scrolling or minimal scrolling.

**Comment by ZakJudges** [comment]
It's a good point. I'll keep this issue open as an enhancement, and we can see what the team thinks in the future. We might even be able to make that section resizable
